<ui version="4.0" >
 <class>KoDetailsPaneBase</class>
 <widget class="QWidget" name="KoDetailsPaneBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DetailsPaneBase</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QSplitter" name="m_splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="m_documentList" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
      <property name="itemsExpandable" >
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="" >
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="3" >
        <widget class="QLabel" name="m_previewLabel" >
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3" >
        <widget class="QLabel" name="m_titleLabel" >
         <property name="font" >
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>81</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <widget class="QCheckBox" name="m_alwaysUseCheckBox" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="toolTip" >
          <string>Always use this template at application start up</string>
         </property>
         <property name="text" >
          <string>Always use this template</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>81</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="3" >
        <widget class="Line" name="line1" >
         <property name="frameShape" >
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3" >
        <widget class="KTextBrowser" name="m_detailsLabel" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>461</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="KPushButton" name="m_openButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="autoDefault" >
      <bool>true</bool>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
  <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ktextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_documentList</tabstop>
  <tabstop>m_alwaysUseCheckBox</tabstop>
  <tabstop>m_detailsLabel</tabstop>
  <tabstop>m_openButton</tabstop>
 </tabstops>
 <includes/>
 <resources/>
 <connections/>
</ui>
